### ISSUE TITLE
New configuration parameter flyway.defaultSchema

### ISSUE THREAD

**Issue by juliahayward** [flyway/flyway#2576]
At the moment the config parameter `flyway.schemas` is overloaded:

- it provides a list of schemas to be created and to be cleaned
- the first of the list is taken to be the default schema for the database connection (where the database supports the concept of a default schema) and the location of the schema history table

We will introduce a new configuration parameter `flyway.defaultSchema` which will take over the second of those two functions. Initially, in Flyway 6.1 and subsequent 6.x.x releases, if `defaultSchema` is not specified then we default back to the current behaviour. In Flyway 7 it is intended that it will become mandatory to specify `flyway.defaultSchema` if `flyway.schemas` is specified.

**Comment by juliahayward** [comment]
Implemented in 6.1.0

**Comment by mlanglet** [comment]
Hi. We're using spring boot 2.2.2 with flyway 6.1.4 and specify spring.flyway.schemas=my_schema and I saw this flyway warning being printed about the defaultSchema property on startup. I tried adding spring.flyway.defaultSchema=my_schema to suppress it but the warning is still showing up. Is this something you peeps at Redgate can fix or is it something spring boot needs to get up to speed with and propagate the property properly?
